asm/compiler.asm

```diff
@@ -1,4 +1,4 @@
-; C, , ; IMMEDIATE [ ] STATE : HEADER LIT LITC COMPILE, LITERAL HERE DODOES
+; included into durexforth.asm
```

Comment:
These word lists are a summary of file contents. I would like to keep them at top of file, because then I can open a file and quickly see what it is about.

I would also rather avoid the "included into" comments. That information is already known, by looking at the place where the files are included. Repeating that goes against the "Don't repeat yourself" principle.

Reply:
To play devil's advocate: by that token the word lists are also repeating yourself. You can find that information by grepping BACKLINK.

I had two thoughts as a new reader. One, I wasn't sure where to start reading. And two, when I see a label in some source file and I don't know where it's from, it'd be nice to have a single index in one file instead of needing to grep all the files. Maybe a developer should be expected to be able to grep.

In any case several of them were out of date. Extra words, missing words.

Reply:
All add a commit to revert and update the lists.